lib/Twig/Test/IntegrationTestCase.php

```diff
@@ -25,6 +25,14 @@
     abstract protected function getFixturesDir();
 
     /**
+     * @return Twig_RuntimeLoaderInterface[]
+     */
+    protected function getRuntimeLoaders()
```

Comment:
I think that Twig is following symfony BC promise.

| If you extend the class and... | Then we guarantee BC... |
|---|---|
| Add a new method | No |

http://symfony.com/doc/current/contributing/code/bc.html